hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/FactoryIdHelper.java

```diff
@@ -112,6 +112,11 @@
 
     public static final String MANAGEMENT_DS_FACTORY = "hazelcast.serialization.ds.management";
     public static final int MANAGEMENT_DS_FACTORY_ID = -36;
+
+    // DataSerializerHook for test IdentifiedDataSerializable classes
+    public static final String TEST_DS_FACTORY = "hazelcast.test.ds.management";
+    public static final int TEST_DS_FACTORY_ID = -37;
+
```

Comment:
Hmm, this is a test code in production code-base.
What I do in tests with IDS classes is I just register the test-factory in the serialization config. 
Maybe it would be a better way to go her too?